gui/src/2D/controls/control.ts

```diff
@@ -37,19 +37,19 @@ export class Control {
     public parent: Nullable<Container>;
     /** @hidden */
     public _currentMeasure = Measure.Empty();
-    private _fontFamily = "Arial";
-    private _fontStyle = "";
-    private _fontWeight = "";
-    private _fontSize = new ValueAndUnit(18, ValueAndUnit.UNITMODE_PIXEL, false);
-    private _font: string;
+    protected _fontFamily = "Arial";
```

Comment:
I am afraid this would bypass some of the font cache mecanism in place , is it intented ?

Reply:
same for all the protected changes

Reply:
up @cronvel

Reply:
Ok it's now back to what it used to be on last push.